pom.xml

```diff
@@ -149,7 +149,7 @@
     <scala.binary.version>2.10</scala.binary.version>
     <jline.version>${scala.version}</jline.version>
     <jline.groupid>org.scala-lang</jline.groupid>
-    <jackson.version>1.8.8</jackson.version>
+    <jackson.version>1.9.13</jackson.version>
```

Comment:
We have to be a little careful about this, since this will affect Jackson versions for all builds. I took a look, with `mvn -Pyarn -Dhadoop.version=2.2.0 -Dverbose dependency:tree | grep org.codehaus.jackson`, and it looks like even this older version of Hadoop brings in generally 1.9.x versions of Jackson. Confusingly, I see Hadoop bringing in 1.8.3, 1.9.11, and 1.9.13, I believe. Going back further, `mvn -Dverbose dependency:tree | grep org.codehaus.jackson` suggests that there was some use of Jackson 1.0.1 in ancient Hadoop 1.0.4.

No matter what, there are Jackson version inconsistencies. Jackson is fairly backwards compatible, so it appears that for example we've gotten away with managing the version up to at least 1.8.8. I'm inclined to say just go with this, indeed, to better match even moderately old Hadoop distributions.

Certainly worth testing a few combinations!